src/components/backend-ai-session-launcher.ts

```diff
@@ -917,7 +917,12 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
             appOptions['runtime'] = 'jupyter';
             appOptions['filename'] = this.importFilename;
           }
-          globalThis.appLauncher.showLauncher(appOptions);
+          // only launch app when it has valid service ports
+          if (service_info.length > 0) {
+            globalThis.appLauncher.showLauncher(appOptions);
+          } else {
+            throw { "message": _text('session.CreationFailed')}
+          }
```

Comment:
In creating a kernel, session-launcher poses a [timeout (10 s)](https://github.com/lablup/backend.ai-console/blob/main/src/components/backend-ai-session-launcher.ts#L961) to wait for the return data from the server. However, there are cases where a session is running after 10 s. For example, there may be a bootstrap scripts which runs longer than 10 s. In those cases, it is inappropriate to display session creation is failed. In my opinion, it would rather better not to display anything.

Reply:
Hmm, I thought that without notification, if the session was immediately removed from the running tab or moved to the exit tab, the user could be confused as to whether the session was successfully created or deleted.
However, I think this part could be solved in the other issue (#772), so I'll fix it as you suggested.